examples/generic-azure.yaml

```diff
@@ -164,12 +164,37 @@ groups:
             - flag: "udpu"
         remediation: |
           ## Remediation
-          It will be difficult to change the corosync MCAST transport to UCAST
-          on an existing cluster. Besides adding the 'transport: udpu' parameter one
-          would have to configure each node separatelly. The easiest way to
-          make the cluster UCAST it to bootstrap it. When creating the cluster
-          you would have to add the key -u, for example
-          $ crm cluster init -u ...
+          To change the corosync MCAST transport to UCAST edit the /etc/corosync/corosync.conf
```

Comment:
So test id 1.1.6 checks if "Corosync transport parameter is set to udpu" , so how does that relate to documenting how to fix the MCAST/UCAST setting? It seems there are actually two separate tests at hand here:
1) Is udpu set correctly, and if not, how to remediate
2) Is UCAST set, and if not how to remediate.   The MCAST/UCAST text is improved, but that remediation belongs in the UCAST check (which I can't find in the xaml file, maybe needs creating?